ansible/roles/riakcs/sync_keys/tasks/main.yml

```diff
@@ -15,3 +15,21 @@
     state: present
   notify: "restart {{ name }}"
   when: "'riak_key' in hostvars[riakcs_control] and 'riak_secret' in hostvars[riakcs_control]"
+
+- name: Update access_id in riakcs datadog agent config
+  lineinfile:
+    dest: "/etc/dd-agent/conf.d/riakcs.yaml"
+    regexp: "([ -]+access_id: ).*$"
+    line: "\\1{{ hostvars[riakcs_control].get('riak_key', 'not-set') }}"
+    backrefs: yes
+  when: "name == 'riakcs' and 'riak_key' in hostvars[riakcs_control] and 'riak_secret' in hostvars[riakcs_control]"
+  notify: restart datadog
+
+- name: Update access_secret in riakcs datadog agent config
+  lineinfile:
+    dest: "/etc/dd-agent/conf.d/riakcs.yaml"
+    regexp: "([ -]+ access_secret: ).*$"
+    line: "\\1{{ hostvars[riakcs_control].get('riak_secret', 'not-set') }}"
+    backrefs: yes
+  when: "name == 'riakcs' and 'riak_key' in hostvars[riakcs_control] and 'riak_secret' in hostvars[riakcs_control]"
+  notify: restart datadog
```

Comment:
I think if we ever want to install a node without datadog, we can fix it then? If it's easy to add the conditional I don't think it'd hurt, but this doesn't offend me too much, unless we're already *not* installing datadog in some environments. (Not sure, for example, if staging, india (softlayer), etc. have datadog installed.)